components/developer/groovy/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		groovy
+# Commit timestamp and previous known release (none here)
+COMPONENT_VERSION=	2.4.7
+COMPONENT_REVISION=	1
+COMPONENT_ARCHIVE=	apache-$(COMPONENT_NAME)-binary-$(COMPONENT_VERSION).zip
+COMPONENT_FMRI=		developer/$(COMPONENT_NAME)
```

Comment:
Not sure about this... In particular, I don't really know the inner workings of the project, like what is the minimally sufficient set of JARs to be the runtime. And it is not that large for it to really matter (although yes, delivering 6Mb vs. 30Mb is yummy).